default.yml

```diff
@@ -126,7 +126,6 @@ Style/IfUnlessModifier:
   Description: Favor modifier if/unless usage when you have a single-line body.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier
   Enabled: false
-  MaxLineLength: 80
```

Comment:
@gizotti , what is these two MaxLength changes, is it related to this PR ?

Reply:
`MaxLineLenght` property was removed from `Style/IfUnlessModifier` and `Style/WhileUntilModifier` cops on `rubocop 0.5.2`